blog post added about the research paper using galaxy workflow (#3185)
galaxyproject/galaxy-hub: content/news/2025-07-crbeam-GRB221009A/index.md

---
title: 'Unraveling the Mystery of GRB 221009A: Secondary Gamma Rays and Cosmic Magnetic Field'
date: "2025-07-01"
tease: "The possibility of multi-TeV secondary gamma rays from GRB221009A"
hide_tease: false

> tags: [esg-wp5, esg]
> subsites: [all-eu, esg]

Reviewer: ```suggestion
tags: [esg-wp5, esg, paper, workflow, workflowhub]
subsites: [all]
```

Reviewer: You can include supporters to your posts which will be shown at the bottom of the page as "supporters" similar to this:
https://galaxyproject.org/events/2025-02-27-research-data-management-with-rspace/
You can find the list of all possible supporters here: 
https://github.com/galaxyproject/galaxy-hub/blob/e3e00255ea3af82b2d236a60d8935c7ba2e6d809/content/supporters.yml

If the organization that you are interested in is not listed there, please add it to that file.

main_subsite: eu
---

> The brightest gamma-ray burst (GRB) ever recorded, GRB 221009A, has puzzled astronomers with its unexpected flux of multi-TeV gamma rays. These high-energy photons, observed at a redshift of 0.151, may challenge conventional models because they should have been absorbed by extragalactic background light (EBL) during their journey across the universe, unless the EBL level is low enogh. A recent [study](https://inspirehep.net/literature/2784795) proposes an elegant solution which doesn't impose any constraints on EBL: these gamma rays may not originate directly from the GRB but are instead secondary particles produced by ultra-high energy cosmic rays interacting with photon backgrounds along the line of sight.  

Reviewer: ```suggestion
The brightest gamma-ray burst (GRB) ever recorded, GRB 221009A, has puzzled astronomers with its unexpected flux of multi-TeV gamma rays. These high-energy photons, observed at a redshift of 0.151, may challenge conventional models because they should have been absorbed by extragalactic background light (EBL) during their journey across the universe, unless the EBL level is low enogh. A recent [study](https://doi.org/10.48550/arXiv.2405.05402) proposes an elegant solution which doesn't impose any constraints on EBL: these gamma rays may not originate directly from the GRB but are instead secondary particles produced by ultra-high energy cosmic rays interacting with photon backgrounds along the line of sight.  
```

Reviewer: Is it okay to use the DOI instead of link for greater permanence and reliability?

 
The study argues that cosmic rays (protons) accelerated by the GRB jet interact with EBL and cosmic microwave background (CMB) photons during their travel. These interactions produce secondary gamma rays through processes like pion decay and pair production. Unlike primary gamma rays, which are heavily absorbed, secondary gamma rays are generated closer to Earth, avoiding significant attenuation.  

> To model these processes, the researchers used the publicly available numerical code [CRbeam](https://github.com/okolo/mcray/tree/main/src/app/crbeam), which simulates cosmic-ray propagation, electromagnetic cascades, and deflections in magnetic fields. This powerful tool is accessible for broader scientific use via the [MMODA astrophysics platform] (https://www.astro.unige.ch/mmoda/node#crbeam) and the [EuroScienceGateway Galaxy workflow] (https://workflowhub.eu/workflows/1369?version=1), enabling researchers to reproduce and extend these findings.

Reviewer: ```suggestion
To model these processes, the researchers used the publicly available numerical code [CRbeam](https://github.com/okolo/mcray/tree/main/src/app/crbeam), which simulates cosmic-ray propagation, electromagnetic cascades, and deflections in magnetic fields. This powerful tool is accessible for broader scientific use via the [MMODA astrophysics platform](https://www.astro.unige.ch/mmoda/node#crbeam) and the [EuroScienceGateway Galaxy workflow](https://workflowhub.eu/workflows/1369?version=1), enabling researchers to reproduce and extend these findings.
```
